debian/control

```diff
@@ -318,9 +318,6 @@ Depends:
 # For "vpn openconnect"
   ocserv,
 # End "vpn openconnect"
-# For "system flow-accounting"
-  pmacct (>= 1.6.0),
```

Comment:
Do not forget to remove mentions about `pmacct` (after merging) in the [vyos-build](https://github.com/vyos/vyos-build/blob/1a6747b038ea49c78b801362ffbabf18eb09543a/data/live-build-config/hooks/live/18-enable-disable_services.chroot#L13)

Reply:
Thanks you, I missed that!
Here is PR: https://github.com/vyos/vyos-build/pull/1030
I'll make it non-draft when ipt_NETFLOW is merged.